resources/langs/nheko_eo.ts

```diff
@@ -0,0 +1,2004 @@
+    </message>
+    <message>
+        <location line="+33"/>
+        <source>%1 was invited.</source>
+        <translatorcomment>%1 estis invitata.</translatorcomment>
+        <translation></translation>
+    </message>
+    <message>
+        <location line="+11"/>
+        <source>%1 changed their display name and avatar.</source>
+        <translation>%1 ŝanĝis sian montriĝan nomon kaj avataron.</translation>
+    </message>
+    <message>
+        <location line="+2"/>
+        <source>%1 changed their display name.</source>
+        <translation>%1 ŝanĝis sian montriĝan nomon.</translation>
+    </message>
+    <message>
+        <location line="+2"/>
+        <source>%1 changed their avatar.</source>
+        <translatorcomment>%1 ŝanĝis sian avataron.</translatorcomment>
+        <translation></translation>
+    </message>
+    <message>
+        <location line="+2"/>
+        <source>%1 changed some profile info.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+4"/>
+        <source>%1 joined.</source>
+        <translation>%1 aliĝis.</translation>
+    </message>
+    <message>
+        <location line="+9"/>
+        <source>%1 rejected their invite.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+2"/>
+        <source>Revoked the invite to %1.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>%1 left the room.</source>
+        <translation>%1 foriris la ĉambron.</translation>
+    </message>
+    <message>
+        <location line="+2"/>
+        <source>Kicked %1.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+2"/>
+        <source>Unbanned %1.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+14"/>
+        <source>%1 was banned.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="-11"/>
+        <source>%1 redacted their knock.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="-943"/>
+        <source>You joined this room.</source>
+        <translation>Vi aliĝis ĉi tiun ĉambron.</translation>
+    </message>
+    <message>
+        <location line="+945"/>
+        <source>Rejected the knock from %1.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+2"/>
+        <source>%1 left after having already left!</source>
+        <comment>This is a leave event after the user already left and shouldn&apos;t happen apart from state resets</comment>
+        <translation type="unfinished">%1 foriris post jam forirado</translation>
```

Comment:
Missing the `!`?

Reply:
Notice: type="unfinished"

Reply:
Still, in this case it is only missing the !, isn't it?

Reply:
No, it only goes to the "... user already left".

Reply:
Plus I'm not sure if what I have is even right :/

Reply:
Ah, I see :D